### PR TITLE
Feature/form edits 2023

### PR DESCRIPTION
**Pull request by rxjeong**
1.adding FEATURE_SERVICE_URL into configurations.ini
2.adding start and completion date into receiving site
3.commenting out attributes not be in AGOL

### CODE REVIEW

File: csv_writer.py [hunk 1]
```diff
@@ -14,18 +14,18 @@ def site_csv_writer(source_sites, receiving_sites, hv_sites):
     """Create source, receive, high volume receiving CSV file"""
     logging.info('Creating soil source site CSV...')
     with open(constant.SOURCE_CSV_FILE, 'w', encoding='UTF8', newline='') as source_file:
-        writer = csv.DictWriter(source_file, fieldnames=constant.SOURCE_SITE_HEADERS)
+        writer = csv.DictWriter(source_file, fieldnames=constant.SOURCE_SITE_HEADERS, extrasaction='ignore')
```
Comment (mishraomp) [inline csv_writer.py:17]: could you please fix the typo for `extraction`

File: helper.py [hunk 2]
```diff
@@ -944,19 +947,15 @@ def map_source_receiving_site_address(_source_sites, _receiving_sites):
...
-    elif _ches_response.status_code == 401:
-        logging.error(constant.CHES_HEALTH_401_STATUS)
-    elif _ches_response.status_code == 403:
-        logging.error(constant.CHES_HEALTH_403_STATUS)
-    else:
-        logging.error("CHES Health returned staus code:%s, text:%s", str(_ches_response.status_code), _ches_response.text)
-    """        
\ No newline at end of file
+    try:
+        _service_response = requests.request("GET", FEATURE_SERVICE_URL, timeout=5)
```
Comment (mishraomp) [inline helper.py:951]: could we rename the `timeout` to be more precise like if it is in millis `timeoutInMillis` or if seconds `timeoutInSeconds`

**Review by mishraomp — APPROVED** [review]
LGTM, some minor feedbacks.

File: csv_writer.py (hunk 1 above)
Comment (rxjeong) [reply csv_writer.py:17]: This(extrasaction='ignore') is not typo. Please see this instruction:
image
https://docs.python.org/3.9/library/csv.html#csv-fmt-params

Comment (mishraomp) [reply csv_writer.py:17]: thanks for the links

File: helper.py (hunk 2 above)
Comment (rxjeong) [reply helper.py:951]: The 'timeout' is fixed parameter name in requests.request. We can't change it. Instead, I put a comment beside of that shows it is timeout in seconds.
`_service_response = requests.request("GET", FEATURE_SERVICE_URL, timeout=5) # timeout in seconds`

Comment (mishraomp) [reply helper.py:951]: thanks that helps :)

**Comment by mishraomp** [comment]
LGTM, please merge and deploy as per your convenience @rxjeong

**Pull request merged**